src/ui.cpp

```diff
@@ -878,6 +878,8 @@ void UpdateDialog::StateUpdateAvailable(const Appcast& info, bool installAutomat
     // take some time to load the MSIE control:
     if ( showRelnotes )
         ShowReleaseNotes(info);
+
```

Comment:
Nitpicking: This appears to use tabs instead of 4 spaces (and would seem to add trailing whitespace on this line too).

Reply:
Hah, been caught by this before. I'll fix